### DISCUSSION TITLE
Example Discord-Notion integration

### DISCUSSION THREAD

**Discussion by varshav0119** [ramnes/notion-sdk-py#19] [Ideas]
Hi,
We're working on a Discord bot for a book club that picks up book titles from a Discord channel and adds them with some details to a Notion database, for all the book club members to access any time they want recommendations!
Turns out the set-up for this is quite simple actually, and we were thinking this could serve as a good example for your notion-sdk-py? notion-sdk-js has similar [examples](https://github.com/makenotion/notion-sdk-js/tree/main/examples) as well.

Can I raise a PR for this?

**Comment by ramnes** [comment]
Yeah, sounds good!
